web3.js/src/connection.ts

```diff
@@ -3603,13 +3610,11 @@ export class Connection {
   }
 
   confirmTransaction(
-    strategy:
-      | BlockheightBasedTransactionConfirmationStrategy
-      | DurableNonceTransactionConfirmationStrategy,
+    strategy: TransactionConfirmationStrategy,
     commitment?: Commitment,
   ): Promise<RpcResponseAndContext<SignatureResult>>;
 
-  /** @deprecated Instead, call `confirmTransaction` using a `TransactionConfirmationConfig` */
+  /** @deprecated Instead, call `confirmTransaction` and pass in {@link TransactionConfirmationStrategy}  @ */
```

Comment:
What does the trailing `@` do here? Was that an accident?

Reply:
ah typo. Fixed.